pkgs/tools/filesystems/sasquatch/default.nix

```diff
@@ -1,76 +1,65 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, fetchurl
-, xz
+, lz4
 , lzo
+, which
+, xz
 , zlib
 , zstd
-, lz4
-, lz4Support ? false
+, bigEndian ? false
 }:
 
 let
-  patch = fetchFromGitHub
-    {
-      # NOTE: This uses my personal fork for now, until
-      # https://github.com/devttys0/sasquatch/pull/40 is merged.
-      # I, cole-h, will keep this fork available until that happens.
-      owner = "cole-h";
-      repo = "sasquatch";
-      rev = "6edc54705454c6410469a9cb5bc58e412779731a";
-      sha256 = "x+PuPYGD4Pd0fcJtlLWByGy/nggsmZkxwSXxJfPvUgo=";
-    } + "/patches/patch0.txt";
-in
-stdenv.mkDerivation rec {
-  pname = "sasquatch";
-  version = "4.4";
+  drv = stdenv.mkDerivation rec {
+    pname = "sasquatch${lib.optionalString bigEndian "-v4be"}";
```

Comment:
This repeats the version number (v4), that's already appended to pname, so the full name would be `sasquatch-v4be-4.5.1.-4`. Perhaps it should just be `sasquatch-be`?

Reply:
The `v4be` indicates the squashfs [filesystem version](https://docs.kernel.org/filesystems/squashfs.html) while the `4.5.1-4` indicates the squashfs-tools version sasquatch is based on. These versions are unrelated.